### PR TITLE
Features/search results links 66

### PR DESCRIPTION
**Pull request by djarredon**
Turns the View Profile button on the search results page into links to the appropriate pages. i.e., organizations to organization profile page, programs to program profile page, and professional to roadmap view.

### CODE REVIEW

File: client/src/components/searchResults/rows.js [hunk 1]
```diff
@@ -15,6 +16,7 @@ const getAvatar = url => {
 
 const tableRow = profile => {
   const { id, type, profileName, avatar, info, contact } = profile
+  console.log('profile type: ', type)
```
Comment (mballeza) [inline client/src/components/searchResults/rows.js]: Remove console statement

File: client/src/modules/reducer/organization.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ export default createReducer(initialState, handlers)
 
 function requestOrganization(state, data) {
   const { payload: { organization } } = data
-  const { media, users } = organization
+  const { media } = organization
```
Comment (mballeza) [inline client/src/modules/reducer/organization.js]: This was removed in an earlier merge. There was a mediaReducer function for the list of users in the organization. Now there are no images in the user table. See https://github.com/iandusenbury/tableland/pull/65/files and add it back.

Comment (djarredon) [reply client/src/modules/reducer/organization.js]: Re-added the function and pushed up the fix.

**Review by mballeza — APPROVED** [review]

File: client/src/components/searchResults/rows.js [hunk 3]
```diff
@@ -23,9 +24,33 @@ const tableRow = profile => {
...
         <div className="search-table-icon search-table-border">
           <div className="search-table-avatar">{avatar}</div>
-          <FlatButton className="search-table-flatbutton" hoverColor="#e7e0d7">
-            View profile
-          </FlatButton>
+          {type === 'User' &&
+            <FlatButton 
+              className="search-table-flatbutton" 
+              hoverColor="#e7e0d7"
+              containerElement={<Link to={`/roadmap/${id}`} />}
```
Comment (iandusenbury) [inline client/src/components/searchResults/rows.js:31]: has this route been added already?

Comment (djarredon) [reply client/src/components/searchResults/rows.js:31]: Yeah, the /roadmap/id route was added with the display_roadmaps_80 branch was merged.

**Pull request merged**